backend/pom.go

```diff
@@ -0,0 +1,129 @@
+// Copyright Amazon.com Inc or its affiliates and the project contributors
+// Written by James Shubin <purple@amazon.com> and the project contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// We will never require a CLA to submit a patch. All contributions follow the
+// `inbound == outbound` rule.
+//
+// This is not an official Amazon product. Amazon does not offer support for
+// this project.
+
+// TODO: should this be a subpackage?
+package backend
+
+import (
+	"context"
+	"encoding/xml"
+	"sort"
+
+	"github.com/awslabs/yesiscan/interfaces"
+	"github.com/awslabs/yesiscan/util/errwrap"
+	"github.com/awslabs/yesiscan/util/licenses"
+)
+
+const (
+	// PomFilename is the file name used by the pomfiles.
+	PomFilename = "pom.xml"
+)
+
+// Pom is a backend for Pom or Project Object Model files. It is an xml file
+// commonly used by the Maven Project under the name pom.xml. We are getting
+// license name by parsing the pom.xml file.
```

Comment:
nit: `the`

Reply:
Fixed, added "the" between getting at the end of line 41 and license at the beginning of line 42 and made the name in line 42 into names.